action.yml

```diff
@@ -8,6 +8,10 @@ inputs:
     description: 'Additional metadata files that should be copied to the root of the deployment package. File patterns are supported, one pattern per line.'
     required: false
     default: ''
+  meta:
```

Comment:
Implementation seems right but I think this name is unnecessarily vague and confusing. If Ramon and I hadn't spoken about this yesterday and I saw `meta: true` on a workflow file I wouldn't have any idea what that meant.

How about `throw-if-no-assets-found` with default `true` instead?